modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-by-using-the-ocp-web-console_{context}"]
+= Configuring NotificationsConfiguration (CR) by using the {OCP} web console
```

Comment:
Again, I would just make sure that the ID and title are the same.

Reply:
Fixed.